### DISCUSSION TITLE
Announcing reNgine-ng (Next Generation) - a detached reNgine fork focused on stability, fast development and co-operation

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Security-Tools-Alliance/rengine-ng#136] [Announcements]
You may know @psyray and me (@AnonymousWP). We have been actively contributing to reNgine since about a year. I used reNgine a lot for my thesis/research in 2023, and due to bugs, missing features and mediocre documentation, I decided to contribute. However, at some point I submitted so many pull requests, and Psyray as well (he has done an excellent job on fixing bugs) that at some point it felt like the efforts were not useful at all. After all, Yogesh is the only maintainer.

Psyray and I shortly joined Yogesh's team, but due to required approvals and inactivities, the development still stagnated. **So in the past months we worked hard behind the scenes on setting up reNgine-ng and fixing bugs. We can finally announce [our first (detached) version: 2.0.7](https://github.com/Security-Tools-Alliance/rengine-ng/releases/tag/v2.0.7)!🥳** In the coming time, we'll be working on [reNgine-ng 2.1.0](https://github.com/Security-Tools-Alliance/rengine-ng/milestone/1).

One of the main reasons why we wanted to start reNgine-ng - and then not just a repository on a single user's account. We did not want to make ourselves dependent when developing, so we decided to create a GitHub Organization. The main benefit is that we can add maintainers as team member to this organization. Furthermore, if either of the owners (Psyray or me) quit developing, we can transfer ownership to others. Open-source projects should never depend on [only that one person](https://xkcd.com/2347/), if you know what I mean.😉

reNgine-ng focuses on the following aspects:

- More stability, thanks to multiple members being able to test the branch before releasing
- A fast development cycle, thanks to multiple members being able to approve pull requests
- Co-operation, thanks to having a group of members, as well as active members on Discord. We're active on Discord for support, too.

Do you wish to contribute or get added to the team? Make sure to show your motivation by fixing some bugs and we'll be glad to add you as a maintainer. You can also contact us through Discord (see our README). Make sure to read [our contribution guidelines](https://github.com/Security-Tools-Alliance/rengine-ng/blob/master/CONTRIBUTING.md).

**Comment by zinwelzl** [comment]
What it easy way to run both rengine and rengine-ng on same server?

**Reply by AnonymousWP** [reply]
You'll likely have to use another port for either of the two services. That way, you can run it on the same IP-address, just a different port.

**Reply by psyray** [reply]
> What it easy way to run both rengine and rengine-ng on same server?

It's not recommended to run on the same VM, 2 VMs is better.
But why use reNgine, use reNgine-ng ;),

**Reply by AnonymousWP** [reply]
Yep, and it won't work as container names are the same.
